TG-4  TG-10

evelynda1985 opened the pull request:
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructo para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructo para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

jramirezl commented on src/receta/Carne.java:
```diff
@@ -11,6 +11,11 @@
+    
+    public Carne(){
+        super();
+        setNombre("Carne");
```
Que bien la llamda del super()
creo que el nombre de la carne deberia ser por ejemplo chuleta de cerdo,

no se como lo podrias pasar si ya estas pasando carne

jramirezl commented on src/receta/Condimento.java:
```diff
@@ -10,5 +10,8 @@
 public class Condimento extends Solido{
-    
+    public Condimento(){
+        super();
```
Si queiro crear el ingrediente pimienta como hago?

jramirezl commented on src/receta/FrutayVerdura.java:
```diff
@@ -11,6 +11,11 @@
+    
+    public FrutayVerdura(){
+        super();
+        setNombre("FrutayVerdura");
```
Si queiro crear zanahoria como hago?

jramirezl commented on src/receta/Liquido.java:
```diff
@@ -11,4 +11,9 @@
+    
+    public Liquido(){
+        super();
+        setNombre("Liquido");
```
Si queiro crear leche como hago

jramirezl commented on src/receta/controller/Main.java:
```diff
@@ -4,75 +4,75 @@
+    hospital.setAliasCocina("Primer piso");    
+    
+    Carne ihospital1 = new Carne();
+    ihospital1.setNombre("Pollo");
```
Bueno aca si estas pasando el nombre Bien , entonces revisa el constructor, podrias crear un constuctor para pasar el nombre :)

evelynda1985 replied:
💃 :D gracias teacher Juan :D. Enviado

evelynda1985 replied:
Pregunta... tengo que poner getters and setters para todas las variables privadas? por ejemplo en el caso de los paramentros debo ponerlos tambien?

jramirezl reviewed (changes requested):
Hola revisa los comentarios, super bien!!!

solo corrige lo q te comente

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructor con parametros tipodelIngrediente (para saber que tipo de carne, condimentos, liquidos, etc tiene el usuario) para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

jramirezl commented on src/receta/FrutayVerdura.java:
```diff
@@ -11,6 +11,12 @@
+    
+    public FrutayVerdura(String tipoFrutayVerdura){
+        super();
+        this.tipoFrutayVerdura = tipoFrutayVerdura;
```
creo q lo que se quiere:
 this.setName (tipoFrutayVerdura);

evelynda1985 replied:
ahhhhhhhhhhhhhh :$

jramirezl commented on src/receta/FrutayVerdura.java:
```diff
@@ -11,6 +11,12 @@
  */
 public class FrutayVerdura extends Solido{
     private boolean contable;
+    private String tipoFrutayVerdura;
```
remover esta variable

jramirezl commented on src/receta/controller/Main.java:
```diff
@@ -4,75 +4,71 @@
+    iUniversidad3.setCantidad(100);
+    iUniversidad3.setMedida("Onzas");
+    
+    }
```
Recuerda agregar cada ingrediente a la cocina

jramirezl reviewed (changes requested):
ya casiiiii

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructor con parametro tipodelIngrediente (el cual hace escritura sobre el nombre del parametro Ingrediente) para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructor con parametro tipodelIngrediente (el cual hace escritura sobre el nombre del parametro Ingrediente) para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

El en main se agrega los ingredientes a la cocina.

jramirezl commented on src/receta/controller/Main.java:
```diff
@@ -4,75 +4,85 @@
+    hospital.setUbicacionCocina("Cincinnati,OH");
+    hospital.setAliasCocina("Primer piso"); 
+    
+    Carne ihospital1 = new Carne("Pollo");
```
recuerdas camel case?

jramirezl reviewed (changes requested):
revisa camel case

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructor con parametro tipodelIngrediente (el cual hace escritura sobre el nombre del parametro Ingrediente) para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

El en main se agrega los ingredientes a la cocina.

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructor con parametro tipodelIngrediente (el cual hace escritura sobre el nombre del parametro Ingrediente) para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

El en main se agrega los ingredientes a la cocina.

Commit:
TG-4  TG-10
En la clase cocina se creo un array tipo Ingrediente.
Se creo un constructor con parametro tipodelIngrediente (el cual hace escritura sobre el nombre del parametro Ingrediente) para cada tipo de Ingrediente Carne, Condimento, FrutayVerdura, Grano, y Liquido.
Se crearon dos lugares en el Main que utiliza varios tipos de ingredientes.

El en main se agrega los ingredientes a la cocina.

Se reviso camelCase, se corrigio y se utilizo peso en la carne de Cerdo en el Main de la cocina universidad.
